controls/models.py

```diff
@@ -215,6 +217,29 @@ def _statement_id_from_control(control_id, part_id):
     def oscal_statement_id(self):
         return Statement._statement_id_from_control(self.sid, self.pid)
 
+    def change_log_add_entry(self, change):
+        # TODO: Test if entry is valid
+        if type(change) is not dict:
```

Comment:
`if not isinstance(change, dict)`
is typically the proper way to do type checks in python

Reply:
Change made.